src/dashboard/src/components/archival_storage/forms.py

```diff
@@ -16,16 +16,50 @@
 # along with Archivematica.  If not, see <http://www.gnu.org/licenses/>.
 
 from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from agentarchives.atom.client import CommunicationError
+from requests import Timeout
+
+from components.archival_storage.atom import get_atom_client
 
 
 class CreateAICForm(forms.Form):
     results = forms.CharField(label=None, required=True, widget=forms.widgets.HiddenInput())
 
+
+class UploadMetadataOnlyAtomForm(forms.Form):
+    slug = forms.CharField(label=_('Insert slug'), required=True)
+    slug.widget.attrs['class'] = 'span8'
+
+    def clean_slug(self):
+        slug = self.cleaned_data['slug']
+        client = get_atom_client()
+        try:
+            client.find_parent_id_for_component(slug)
+        except Timeout as e:
+            raise forms.ValidationError(_('Connection timed out: AtoM server cannot be reached.'))
+        except CommunicationError as e:
+            if '404' in e.message:
+                raise forms.ValidationError(_('Description with slug %(slug)s not found!'), code='notfound', params={'slug': slug})
+            raise forms.ValidationError(_('Unknown error: %(error)s'), code='error', params={'error': e.message})
+        return slug
+
+
 class ReingestAIPForm(forms.Form):
     METADATA_ONLY = 'metadata'
     OBJECTS = 'objects'
     REINGEST_CHOICES = (
-        (METADATA_ONLY, 'Re-ingest metadata only'),
-        (OBJECTS, 'Re-ingest metadata and objects')
+        (METADATA_ONLY, _('Metadata only')),
+        (OBJECTS, _('Metadata and objects'))
     )
-    reingest_type = forms.ChoiceField(choices=REINGEST_CHOICES,  widget=forms.RadioSelect, required=True)
+    reingest_type = forms.ChoiceField(choices=REINGEST_CHOICES, widget=forms.RadioSelect, required=True)
+
+
+class DeleteAIPForm(forms.Form):
+    uuid = forms.CharField(label=_('Please type in the UUID of the repository to confirm'), required=True)
```

Comment:
What is this field looking for? It doesn't validate the input against anything, and appears to discard it unused.